src/Files.App/BaseLayout.cs

```diff
@@ -606,11 +606,10 @@ public async void BaseContextFlyout_Opening(object sender, object e)
                 if (!InstanceViewModel.IsPageTypeSearchResults && !InstanceViewModel.IsPageTypeZipFolder)
                 {
                     var shellMenuItems = await ContextFlyoutItemHelper.GetBaseContextShellCommandsAsync(currentInstanceViewModel: InstanceViewModel, workingDir: ParentShellPageInstance.FilesystemViewModel.WorkingDirectory, shiftPressed: shiftPressed, showOpenMenu: false, shellContextMenuItemCancellationToken.Token);
-                    if (shellContextMenuItemCancellationToken.IsCancellationRequested)
```

Comment:
The old logic for detecting cancellation is invalid because it is always checking the new instanitated cts.